spec.html

```diff
@@ -3138,6 +3138,18 @@ <h1>NormalCompletion</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-throwcompletion" aoid="ThrowCompletion">
+        <h1>ThrowCompletion</h1>
+        <p>The abstract operation ThrowCompletion with a single _argument_, such as:</p>
+        <emu-alg>
+          1. Return ThrowCompletion(_argument_).
+        </emu-alg>
+        <p>Is a shorthand that is defined as follows:</p>
```

Comment:
I don't think there's any need to make it a shorthand. I suppose this is by analogy with NormalCompletion, but I don't think that needs to be a shorthand either.

Reply:
I find NormalCompletion to be awesome at simplifying the spec; what would you suggest instead of both?

Reply:
Sorry, I'm not saying it shouldn't be defined, just that it doesn't need to be defined *as a "shorthand"* (macro). That is, it can be defined as a conventional abstract operation.

Reply:
Ah, sure. Yes, this was just consistency with NormalCompletion; I think changing both is fine but should be done in a followup (or, NormalCompletion should be changed prior to this PR going in).

Reply:
E.g., `ReturnIfAbrupt` *has* to be defined as shorthand/macro, because 'invoking' it can cause a return from the invoking algorithm, which a conventional abstract operation can't do.

Reply:
I'm fine to submit a follow-up to change both.

Reply:
Yay! Then your stamp on this PR would be appreciated ;-)

Reply:
My stamp?

Reply:
A PR approval via Github's PR Reviews feature :-) only if you approve, of course!